### DISCUSSION TITLE
the link "register with xdm" doesn't working

### DISCUSSION THREAD

**Discussion by hereparvezali** [subhra74/xdm#924] [Q&A]
the link "register with xdm" doesn't working

**Comment by subhra74** [comment]
Please try latest version 8.0.18, and let me know if this solves the issue.
https://github.com/subhra74/xdm-experimental-binaries/releases/tag/8.0.18-beta

**Reply by functionalatbest** [reply]
I am having the same issue with the latest release. Oddly enough it worked 1 time and I was able to capture a video, then it suddenly stopped connecting and nothing seems to work
